src/main/java/com/onarandombox/MultiverseCore/commands/DumpsCommand.java

```diff
@@ -157,14 +154,27 @@ public void run() {
     private String getLogs() {
         // Get the Path of latest.log
         Path logsPath = plugin.getServer().getWorldContainer().toPath().resolve("logs").resolve("latest.log");
+        File logsFile = logsPath.toFile();
+
+        if (!logsFile.exists()) {
+            Logging.warning("Could not read logs/latest.log");
+            return "Could not read ./logs/latest.log";
+        }
+
+
 
-        // Try to read file
         try {
-            return Files.readString(logsPath);
+            return Files.readString(logsPath, StandardCharsets.ISO_8859_1);
         } catch (IOException e) {
-            Logging.warning("Could not read logs/latest.log");
-            throw new RuntimeException(e);
+            // UTF-8 encoded log
```

Comment:
Can just be a separate try..catch, don't need to nest them. Bcu if it succeeds it will return already